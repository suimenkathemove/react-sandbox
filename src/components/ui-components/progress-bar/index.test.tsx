import { cleanup, render } from '@testing-library/react';

import { defaultProps } from './index.stories';

import 'jest-styled-components';
import { ProgressBar } from './';

const setup = () => render(<ProgressBar {...defaultProps} />);

describe('ProgressBar', () => {
  afterEach(() => {
    cleanup();
  });

  test('render', () => {
    const { asFragment } = setup();
    expect(asFragment()).toMatchSnapshot();
  });
});
