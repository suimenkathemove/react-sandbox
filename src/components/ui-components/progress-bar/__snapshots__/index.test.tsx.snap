// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar render 1`] = `
<DocumentFragment>
  .c0 {
  width: 100%;
  height: 8px;
  -webkit-appearance: none;
  -moz-appearance: none;
  appearance: none;
}

.c0::-webkit-progress-bar {
  background-color: gray;
  border-radius: 4px;
}

.c0::-webkit-progress-value {
  background-color: red;
  border-radius: 4px;
}

<progress
    class="c0"
    max="2"
    value="1"
  />
</DocumentFragment>
`;
